c#-ectoken/ec_encryptcore/ec_encryptcore.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
@jcline - Thanks for including the c# version with interop tests!

Unfortunately, his change breaks the build on windows. (same for the other cs proj file). Can we exclude the C# version from the interop checks until the new ectoken-csharp repo (preview -https://github.com/hattan/ectoken-csharp) is merged in?

The new repo works cross platform on windows and *nix (using .net core, which is preferred over mono)

Thanks!